framework/auth/core.py

```diff
@@ -374,6 +376,9 @@ class User(GuidStoredObject, AddonModelMixin):
     # user language and locale data (e.g. 'en_US')
     locale = fields.StringField(default='en_US')
 
+    # created_through_osf4m:
+    conference_user = fields.BooleanField(default=False)
```

Comment:
I think renaming this to `is_conference_user` would make it more clear that this is a boolean field and not a user object.